src/ipelib/flip/CTIPE-int.f

```diff
@@ -538,14 +559,22 @@ SUBROUTINE CTIPINT(
 
       !.. electron density for photoelectron routine
       DO J=JMIN,JMAX
-        EDEN(J)=XIONN(1,J)+XIONN(2,J)+XIONN(3,J)+XIONN(4,J)+XIONN(5,J)+
-     >    XIONN(6,J)
+        electron_density(J)=XIONN(1,J)+XIONN(2,J)+XIONN(3,J)+XIONN(4,J)+
+     >                      XIONN(5,J)+XIONN(6,J)
       ENDDO
 
       !.. 2-stream photoelectron routine to get electron heating 
       !.. rate and secondary ion production
-      CALL PE2S(F107,F107A,N,TI,FPAS,-1.0E22,EDEN,UVFAC,COLUM,
-     > IHEPLS,INPLS,INNO,mp,lp)
+

```

Comment:
Could this conditional be parameterized?

Reply:
I think this line is removed. The new call doesn't have the number.